compiler/src/dotty/tools/dotc/interactive/Interactive.scala

```diff
@@ -344,7 +344,7 @@ object Interactive {
     if (tree.pos.contains(pos)) {
       // FIXME: We shouldn't need a cast. Change NavigateAST.pathTo to return a List of Tree?
       val path = NavigateAST.pathTo(pos, tree, skipZeroExtent = true).asInstanceOf[List[untpd.Tree]]
-      path.dropWhile(!_.hasType).asInstanceOf[List[tpd.Tree]]
+      path.dropWhile(!_.hasType) collect { case t: tpd.Tree @unchecked => t }
```

Comment:
If `path` can contain non-tree elements then `hasType` will also throws. Maybe change `NavigateAST.pathTo` to discard non-tree element and return a `List[Tree]` instead (as suggested in the comment).

Reply:
Good point.

Reply:
I made the change on the other PR (harden-ide)